container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -328,10 +329,29 @@ function run_git_tempest {
     tempest init openshift
     pushd $TEMPEST_DIR
 
-    discover_tempest_config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
+    # We're running cleanup only under certain circumstances
+    if [[ ${TEMPEST_DEBUG_MODE} == false ]] && [[ ${TEMPEST_CLEANUP} == true ]]; then
+        # generate a simple tempest.conf so that we can run --init-saved-state
+        discover-tempest-config
+        # let's remove the images that discover-tempest-config creates by default
+        # so that the're not part of the saved_state.json and can be deleted
+        # by tempest cleanup later
+        openstack image delete cirros-0.6.2-x86_64-disk.img
+        openstack image delete cirros-0.6.2-x86_64-disk.img_alt
```

Comment:
If we are indeed going by this approach maybe you can just do something like 
`openstack image list -c Name -f value | xargs -I {} openstack image delete {}` 
That way we won't be hardcoding values  in case image name changes or we use a newer version in future  .

Reply:
This sounds like a good idea! My only concern is that it deletes all images. Maybe we can update the command with grep for cirros?

```
openstack image list -c Name -f value 
   | grep cirros 
   | xargs -I {} openstack image delete {}
```

Reply:
sounds good, let's do that, it's better as we won't need to update it with every cirros version update in tempestconf